pkg/cmd/root.go

```diff
@@ -89,7 +90,21 @@ By default, the default directories depend on the Operating System. The defaults
 | Windows          | %TEMP%\helm               | %APPDATA%\helm                 | %APPDATA%\helm          |
 `
 
-func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string) (*cobra.Command, error) {
+var settings = cli.New()
+
+func Debug(format string, v ...interface{}) {
+	if settings.Debug {
+		format = fmt.Sprintf("[debug] %s\n", format)
+		log.Output(2, fmt.Sprintf(format, v...))
```

Comment:
I realize this was just copied code, but...

```suggestion
		log.Output(os.Stderr.Fd(), fmt.Sprintf("[debug] " + format + "\n",, v...))
```